onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -1923,6 +1923,179 @@ Matrix product that behaves like numpy.matmul: https://docs.scipy.org/doc/numpy-
         FusedMatMulShapeInference(ctx);
       });
 
+    static const char* NonZero_doc = R"DOC(
+  Behaves similar to https://github.com/onnx/onnx/blob/master/docs/Operators.md#NonZero
+  returns a tensor that contains coordinates for non-zero entries. Even though the input
+  is a sparse tensor, that usually does not contain zeros, the model logic may construct
+  sparse tensor containing explicit zeros that convey model specific meaning.
```

Comment:
As per the ONNX spec, zero is a default value (for numeric tensors). Do we plan to not validate this for LinkedIn use cases?

Reply:
By default, Sparse Tensors do not contain zeros at all. However, when a sparse tensor is constructed from user data, we do not enforce that it does not contain zeros. LinkedIn requests this to be a feature. I am fine with it.

Reply:
As an additional context, this information (zero or missing) is important for model correctness. TensorFlow calls this [explicit zeros](https://www.tensorflow.org/guide/sparse_tensor#distinguishing_missing_values_from_zero_values).